src/app/app.scss

```diff
@@ -0,0 +1,38 @@
+@import url(https://fonts.googleapis.com/css?family=Roboto:400,300,500,700);
+@import url(https://fonts.googleapis.com/icon?family=Material+Icons);
```

Comment:
since our site is deployed privately -- we would gain more if we use local stored fonts